README.md

```diff
@@ -1,3 +1,4 @@
-# robot_vision
+# robot_vision_base
 
-This repo contains the ROS implementation of YOLOv3. The output of the algorithm is a bounding box around the detected object. 
+This repository contains the classes for an image publisher and an image subscriber.
```

Comment:
Please include the ROS and python versions required for using this repository. Correct me if I'm wrong, but I think it's:
ROS Noetic (Ubuntu 20.04)
Python 3.8+